sanity/lib/getImage.ts

```diff
@@ -0,0 +1,15 @@
+import { client } from './client';
+import { useNextSanityImage } from 'next-sanity-image';
```

Comment:
Separate external imports from internal imports by adding an empty line between them. External imports should go at the top, followed by internal.